docs/contributing/development.md

```diff
@@ -0,0 +1,198 @@
+
+5. (TODO)
+
+6. (TODO)
+
+You will now be ready to begin building or modifying the project.
+
+### Make changes to the repo
+
+Once you have your repository, you can get to work.
+
+1. Rebase your local branches against upstream `staging` so you are working off the latest changes:
+
+   ```sh
+   git fetch --all
+   git rebase upstream/staging
+   ```
+
+1. Create a local feature branch off of `staging` to make your changes:
+
+   ```sh
+   git checkout -b my-feature staging
+   ```
+
+1. Make your changes and commits to this local feature branch.
+
+1. Repeat step 1 on your local feature branch once you're done your work, to ensure you have no conflicts with other work done since you stated.
+
+1. Push up your local feature branch to your GitHub fork:
+
+   ```sh
+   git push --set-upstream origin my-feature
+   ```
+
+1. On GitHub, create a new PR against the upstream `staging` branch following the advice below.
+
+1. Once your PR is merged, ensure you keep your local branches up-to-date:
+
+   ```sh
+   git fetch --all
+   git checkout staging
+   git rebase upstream/staging
+   git push -u origin staging
+   ```
+
+1. Delete your local feature branch if you no longer need it:
+
+   ```sh
+   git branch -d my-feature
+   ```
+
+### CONTRIBUTORS.md
+
+If it's your first time contributing code to a particular repository, please add yourself to the `CONTRIBUTORS.md` file at the bottom of the `OpenBullet2 Contributors` section. While GitHub does track this, having the written document makes things clearer if the code leaves GitHub and lets everyone quickly see who has worked on the project for copyright or praise!
+
+## Official Branches
+
+### Feature Branches
+
+From time to time, major projects may come up that require multiple PRs and contributions from multiple people. For these tasks, feature branches specific to the feature should be created, based off of `staging`. This helps allow the work to progress without breaking `master` for long periods and allowing those interested in that particular project the ability to work at their own pace instead of racing to fix a broken feature before the next release. To create a feature branch, please communicate with a Core team member and that can be arranged.
+
+Once the feature, a feature branch was created for is ready, it can be merged in one shot into `staging` and the feature branch removed.
+
+### The Master Branch
+
+The `master` branch is the primary face of the project and main branch. Except for emergency release hotfixes the `staging` development branch, all PRs should target `staging`. As a general rule, no PR should break master and all PRs should be tested before merging to ensure this does not occur. We're only human and this is still likely to happen, but you should generally be safe to build off of `stating` if you want the latest and greatest version of OpenBullet2.
```

Comment:
I think it's missing a "should be used" or something after staging, otherwise the sentence does not make sense.

Reply:
Could you make a suggestion of code cause I don't understand where you want "should be used" to be added.